### DISCUSSION TITLE
Adding basic auth to a container

### DISCUSSION THREAD

**Discussion by srz09** [Dokploy/dokploy#562] [Q&A]
Hello,
I want to protect with basic auth plugin one of my container but it looks impossible to do so...
I tried manually to create my middleware in traefik files and add custom labels to my compose but as the compose is then adding extra labels for traefik, it overrides my middleware by the secure middleware...
Any idea how I can do so ?

**Comment by Siumauricio** [comment]
If you really want to do something specific with middlewares and the labels that add the domains are interfering, it would be better to add the labels manually
